values.yaml

```diff
@@ -54,6 +54,8 @@ expose:
   clusterIP:
     # The name of ClusterIP service
     name: harbor
+    # The ip address of the ClusterIP service (leave empty for acquiring dynamic ip)
+    clusterIP: ""
```

Comment:
could you please change `clusterIP.clusterIP` to `clusterIP.staticClusterIP`?
Thanks

Reply:
Done